modules/ROOT/pages/configure-partner.adoc

```diff
@@ -122,6 +140,19 @@ You can delete non AS2 identifiers such as all X12 ISA, X12 GS, and DUNS identif
 
 . If you need a replacement, add it now by clicking *New*.
 
+== Delete Endpoints
+
+You can delete any endpoints that are not used in a message flow configuration.
+
+. From within your Anypoint Partner Manager Sandbox or Production environment, select <host partner name> from the upper left.
+
+. From _Endpoints: Send to <partner name>_ search for the endpoint you want to delete by Name, Protocol, URL, or Description.
++
+If you can’t remember what to search for, scan the list for any endpoints that don’t have a message flow associated with them.
```

Comment:
Earlier text mentions the 0 in the endpoint column?

Reply:
Yes, fixed.